www/src/content/docs/docs/get-started.mdx

```diff
@@ -1,86 +1,319 @@
-      Workers and Node.js. You can upvote support for Bun and Deno on [their
-      respective issues here](https://github.com/fiberplane/fpx/issues/192) to
-      help us prioritize development.
-    </Aside>
+[Zod OpenAPI Hono](https://github.com/honojs/middleware/tree/main/packages/zod-openapi) provides input validation, type safety, and OpenAPI documentation.
+It extends the Hono class and routes have to be defined and passed as parameters in the handlers.
 
-3.  ### Launch Fiberplane Studio
+Install the package alongside hono and zod:
 
-        In a new terminal window start Fiberplane Studio from your project directory:
+<PackageManagers type="add" pkg="hono zod @hono/zod-openapi" />
 
-        <PackageManagers type="exec" pkg="@fiberplane/studio" />
+Create an app using Hono Zod
 
-        Finally, start (or restart) your application, and you're ready to go!
+```ts
+import { OpenAPIHono } from "@hono/zod-openapi";
+const app = new OpenAPIHono<{
+  Bindings: Bindings;
+  Variables: Variables;
+}>();
+```
 
-</Steps>
+Define your input data schemas in Zod:
 
-## Next steps
+```ts
+const UserResponseSchema = z
+  .object({
+    id: z.number().int().openapi({
+      description: "The unique identifier for the user",
+      example: 1
+    }),
 
-<LinkCard
-  title="Learn about the client library"
-  description="Read more about the client library and how to configure it"
-  href="/docs/components/client-library"
-/>
+    name: z.string().min(1).max(255).openapi({
+      description: "The user's full name",
+      example: "Matthew"
+    }),
 
-<LinkCard
-  title="Make HTTP requests from Studio"
-  description="Use the Studio as your API client in your local development"
-  href="/docs/features/making-requests"
-/>
+    email: z.string().email().openapi({
+      description: "The user's email address",
+      example: "matthew@cloudflare.com"
+    })
+  })
+  .openapi("User", {
+    description: "Schema for user response including ID"
+  });
 
-<LinkCard
-  title="Inspect request traces"
-  description="Visualize the entire request-response lifecycle of your API"
-  href="/docs/features/inspect-traces"
-/>
+const CreateUserSchema = UserResponseSchema.omit({ id: true }).openapi(
+  "CreateUser",
+  {
+    description: "Schema for creating a new user"
+  }
+);
+```
 
-<LinkCard
-  title="Generate requests with AI"
-  description="Use language models to generate request data to test your API"
-  href="/docs/features/generating-with-ai"
+<Aside type="note">
+  `.extend` does not propagate the openapi metadata to the new schema. Instead
+  use `.merge` .
+</Aside>
+
+Define your routes in Zod and include the input and output data schemas if needed:
+
+```ts
+const createUser = createRoute({
+  method: "post",
+  path: "/api/user",
+  request: {
+    // Validate request body using Zod schemas
+    body: {
+      required: true, // NOTE: this is important to set to true, otherwise the route will accept empty body
```

Comment:
```suggestion
     // NOTE: this is important to set to true, otherwise the route will accept empty bodies and invalid content types
      required: true,
```